lib/js/src/rpc/RpcCreator.js

```diff
@@ -752,20 +752,6 @@ class RpcCreator {
                     message = new OnEncodedSyncPData(params);
                 }
                 break;
-            case FunctionID.ListFiles:
-                if (rpcType === RpcType.REQUEST) {
-                    message = new ListFiles(params);
-                } else if (rpcType === RpcType.RESPONSE) {
-                    message = new ListFilesResponse(params);
-                }
-                break;
-            case FunctionID.DeleteFile:
-                if (rpcType === RpcType.REQUEST) {
-                    message = new DeleteFile(params);
-                } else if (rpcType === RpcType.RESPONSE) {
-                    message = new DeleteFileResponse(params);
-                }
-                break;
```

Comment:
I don't think these should be removed. Might be out of sync with the `managers` branch?

Reply:
This one was a weird change. There was already a ListFiles and DeleteFile case earlier up in the file before these two. I'm not sure how that happened, but this change removes the duplicate cases.